admin/client/src/styles/legacy/_forms.scss

```diff
@@ -494,8 +494,8 @@ button.loading {
 	&[class*="font-icon-"],
 	&[class^="font-icon-"],
 	&.ss-ui-button-constructive[class*="font-icon-"] {
-		padding: 5px 8px;
-		margin-bottom: $grid-y*1.5;
+		padding: 6px 8px;
```

Comment:
I should create a variable for it actually as its a standard measurement. Same with the line-height, there is already a variable for it. Will rectify.